### PR TITLE
feat: use resultcard in lnurlauth

### PR DESCRIPTION
**Pull request by im-adithya**
### Describe the changes you have made in this PR

Align the success screen of LNURLAuth

### Link this PR to an issue [optional]

#1379

### Type of change

- `feat`: New feature (non-breaking change which adds functionality)

### Screenshots of the changes [optional]
Screenshot from 2022-10-07 21-47-02

### How has this been tested?

Manually

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

File: src/i18n/locales/en/translation.json [hunk 1]
```diff
@@ -476,7 +476,7 @@
         "heading": "Do you want to login to"
       },
       "submit": "Login",
-      "success": "Authenticated successfully",
+      "success": "Authenticated successfully into {{name}}",
```
Comment (escapedcat) [inline src/i18n/locales/en/translation.json:479]: Is "with" better than "into"? @reneaaron

**Review by escapedcat — APPROVED** [review]
ack

File: src/i18n/locales/en/translation.json (hunk 1 above)
Comment (reneaaron) [reply src/i18n/locales/en/translation.json:479]: "Authenticated into" feels wrong to me. How about "Login successful on {{name}}"?

Comment (im-adithya) [reply src/i18n/locales/en/translation.json]: Done!

**Pull request merged**